app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -245,43 +245,38 @@ def build_metric_id(type, resource, metric_id)
     # by ems_ref, which in Hawkular terms is the
     # fully qualified resource path from Hawkular inventory
     def run_generic_operation(operation, ems_ref, parameters = {})
-      with_provider_connection do |connection|
         the_operation = {
           :operationName => operation,
```

Comment:
This is actually a JSON that will be sent to the cmdgw impl in Java, it expects the camel case notation.